src/sensors/icm20948sensor.cpp

```diff
@@ -34,204 +34,93 @@ int bias_save_periods[] = { 120, 180, 300, 600, 600 }; // 2min + 3min + 5min + 1
 // Accel scale conversion steps: LSB/G -> G -> m/s^2
 constexpr float ASCALE_4G = ((32768. / ACCEL_SENSITIVITY_4G) / 32768.) * EARTH_GRAVITY;
 
-// #ifndef ENABLE_TAP
-//     #define ENABLE_TAP false
-// #endif
-
-void ICM20948Sensor::save_bias(bool repeat) {
-#if defined(SAVE_BIAS) && SAVE_BIAS
-    #ifdef DEBUG_SENSOR
-    m_Logger.trace("Saving Bias");
-    #endif
+void ICM20948Sensor::motionSetup() 
+{
+    connectSensor();
+    startDMP();
+    load_bias();
```

Comment:
Could you rename this function to `loadBias()`?

Reply:
I'll rename save_bias too to keep them with the same name structure.